lib/src/workmanager.dart

```diff
@@ -201,6 +201,7 @@ class JsonMapperHelper {
     final bool _isInDebugMode, {
     final int callbackHandle,
   }) {
+    assert(_isInDebugMode != null);
```

Comment:
moet dit? Kan dit niet een optional zijn met een default van false?

Reply:
zou het sowieso named maken ik gruwel van boolean waardes in constructors

Reply:
Sure, I mirrored the other JsonMapperHelper function actually. There it was an unnamed argument, without assert. On the defaulting to false... maybe a null gets translated to false anyway?

Anyway, I'll make it named in both functions and update the tests! 👍